config/packages/messenger.yml

```
@@ -0,0 +1,6 @@
+framework:
+    messenger:
+        default_bus: default
+        buses:
+            default: ~
+            query.bus: ~
```

For me we should distinguish the public api query bus 
```suggestion
            public_query.bus: ~
```